crates/diesel_models/src/unified_translations.rs

```diff
@@ -0,0 +1,64 @@
+//! Translations
+
+use diesel::{AsChangeset, Identifiable, Insertable, Queryable, Selectable};
+use time::PrimitiveDateTime;
+
+use crate::schema::unified_translations;
+
+#[derive(Clone, Debug, Queryable, Selectable, Identifiable)]
+#[diesel(table_name = unified_translations, primary_key(unified_code, unified_message, locale), check_for_backend(diesel::pg::Pg))]
+pub struct UnifiedTranslations {
+    pub unified_code: String,
+    pub unified_message: String,
+    pub locale: String,
+    pub translation: String,
+    pub created_at: PrimitiveDateTime,
+    pub last_modified: PrimitiveDateTime,
+}
+#[derive(Clone, Debug, Insertable)]
+#[diesel(table_name = unified_translations)]
+pub struct UnifiedTranslationsNew {
+    pub unified_code: String,
+    pub unified_message: String,
+    pub locale: String,
+    pub translation: String,
+    pub created_at: PrimitiveDateTime,
+    pub last_modified: PrimitiveDateTime,
+}
+
+#[derive(Clone, Debug, AsChangeset)]
+#[diesel(table_name = unified_translations)]
+pub struct UnifiedTranslationsUpdateInternal {
+    pub unified_code: Option<String>,
+    pub unified_message: Option<String>,
+    pub locale: Option<String>,
+    pub translation: Option<String>,
+    pub last_modified: PrimitiveDateTime,
+}
+
+#[derive(Debug)]
+pub struct UnifiedTranslationsUpdate {
+    pub unified_code: Option<String>,
```

Comment:
We would need not update unified_code, unified_message and locale. Can we remove it from here is not required?